libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -117,6 +117,9 @@ size_t ASTJsonConverter::sourceIndexFromLocation(SourceLocation const& _location
 
 string ASTJsonConverter::sourceLocationToString(SourceLocation const& _location) const
 {
+	if (_location == SourceLocation{})
```

Comment:
How did this work before?

Reply:
We didn't. 

That yielded into source indices with value `numeric_limits<size_t>::max()` and that caused a throw in `std::stoi()` on the decoding side, so I tried to properly handle in the decoding side but realized that it looks like an unclean solution and the `ASTImportTest.sh` kept failing. So I decided for this solution (I think is the shortest & cleaned?)